pyproject.toml

```diff
@@ -0,0 +1,91 @@
+[tool.poetry]
+name = "hyperstyle-analysis-prod"
+version = "0.1.0"
+description = "Repository containing several tools-helpers to work with data from the Hyperstyle platform"
+authors = ["Anastasia Birillo <anastasia.birillo@jetbrains.com>"]
```

Comment:
Please, put yourself also to the authors list

Reply:
I also added Maria Tigina here :-)